docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md

```diff
@@ -176,67 +191,62 @@ After a failover, do the following:
 
```

Comment:
disable HA completely is a little ambiguous. Maybe delete the HA configuration and follow the setup instructions again.